server/answer.go

```diff
@@ -160,7 +159,7 @@ func (qc queueCaller) PipelineRecv(ctx context.Context, transform []capnp.Pipeli
 			return nil
 		}
 		return b.recv(ctx, transform, r)
-	default:
+	} else {
```

Comment:
It seems like the `else` isn't needed here, since the `if` block returns.  I would personally find it even more readable without the `else`.

Reply:
I agree. Force-pushed a version that makes this change.